src/classic/clvm_tools/stages/stage_2/operators.rs

```diff
@@ -52,6 +52,36 @@ pub struct CompilerOperatorsInternal {
     opt_memo: RefCell<HashMap<AllocatorRefOrTreeHash, NodePtr>>,
 }
 
+/// Given a list of search paths, find a full path to a file whose partial name
+/// is given.  If the file can't be found in any search path, use the expression
+/// the user gave to cause the file to be searched for in the error result.
+pub fn full_path_for_filename(
+    parent_sexp: NodePtr,
+    filename: &str,
+    search_paths: &[String],
+) -> Result<String, EvalErr> {
+    for path in search_paths.iter() {
+        let mut path_buf = PathBuf::new();
+        path_buf.push(path);
+        path_buf.push(filename);
+        let f_path = path_buf.as_path();
+        if f_path.exists() {
+            return f_path
+                .to_str()
+                .map(|x| x.to_owned())
+                .map(Ok)
+                .unwrap_or_else(|| {
+                    Err(EvalErr(
+                        parent_sexp,
+                        "could not compute absolute path".to_string(),
```

Comment:
Suggestion: This error message should include `f_path` in the message

Reply:
makes sense